src/lib/radio/radio.spec.ts

```diff
@@ -485,6 +485,28 @@ describe('MdRadio', () => {
 
       expect(fruitRadioNativeInputs[0].getAttribute('aria-labelledby')).toBe('uvw');
     });
+
+    it('should make the native input element a tab stop', () => {
+      expect(fruitRadioNativeInputs[0].tabIndex).toBe(0);
+    });
```

Comment:
Being a tab stop is something better covered by an e2e test where we can actually press the tab key, but it is good to have a unit test that asserts we're forwarding the tabIndex property.